### ISSUE TITLE
Fix "add another X" buttons on ingest form

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#341]
### Descriptive summary

(This may be seen elsewhere, I just happened to notice it in the ingest form)

Some styling has been overridden on the ingest form (I don't know if this is our work or base hyrax styling) such that the "add another X" buttons (a) are not obviously buttons - in fact they appear to be links, and (b) they have a barely-perceptible color change on keyboard focus.

The first issue is a minor red flag, as it can confuse users, but it isn't currently a WCAG failure.  It's worth consideration, however, as it is a known concern in the accessibility community that could eventually become a failure as they dive deeper into accessibility issues that affect cognitively-impaired individuals.  As a simple example, links typically don't respond to the spacebar.  Some browsers / plugins make the spacebar scroll the page down.  If you think you're on a link, you might hit the spacebar expecting the page to scroll.  Another issue is sighted users who use a screen reader for cognitive reasons.  They tab to the button, which they think is a link, and the screen reader defines it as a button.  This is a minor inconvenience, but it can still be a little confusing.

The second issue is a definite failure.  On focus, something more than color must be used to indicate  focus.  Were we to leave the default styling in place for buttons, we could let the browser decide how to handle this, but because we (or Hyrax) chose to override the styling, it's now our responsibility to make sure focus is very clear.

### Expected behavior

When tabbing to one of these buttons, there should be a clear focus indicator, such as an outline around the button.

Ideally, these buttons should look like buttons in some way, not be buttons that are styled to look like links.

**Comment by raybrarian** [comment]
Has any work happened in Hyrax to address this issue?

**Comment by jechols** [comment]
Same as the other issue, unknown:

> I'd have to look.  Maybe slate this to look at for WC12?  I could spin it up "real quick" but that might waste an hour or something.  It's also worth considering Hyrax 3's state since we're migrating there semi-soon I think.

**Comment by CGillen** [comment]
"Add another X" buttons have been changed to buttons (by hyrax or us, I don't know) and now provide a much nicer outline on tab focus.
